Formula/certbot.rb

```diff
@@ -3,8 +3,8 @@ class Certbot < Formula
 
   desc "Tool to obtain certs from Let's Encrypt and autoenable HTTPS"
   homepage "https://certbot.eff.org/"
-  url "https://github.com/certbot/certbot/archive/v1.14.0.tar.gz"
-  sha256 "3e307c461f381065a2f03d153db2842ad2f593b4cfe6ccc4574e2cef8510122b"
+  url "https://github.com/certbot/certbot/archive/v1.16.0.tar.gz"
```

Comment:
Can't use Pypi URL since that is specifically for `certbot`.
This formula installs `certbot, certbot-apache, certbot-nginx, acme`

Reply:
Changed to PyPi URL with `certbot-apache` and `certbot-nginx` as resources instead.